app/src/main/java/com/cmput301f20t21/bookfriends/ui/browse/BrowseDetailActivity.java

```diff
@@ -24,9 +23,9 @@ protected void onCreate(@Nullable Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         vm = new ViewModelProvider(this).get(BrowseDetailViewModel.class);
         actionButton = findViewById(R.id.detail_action_button);
-
         actionButton.setText(R.string.request);
         actionButton.setOnClickListener(this::sendRequest);
+        button.setText(R.string.browse_button);
```

Comment:
there's a duplicate here, I think actionButton can be removed since we already have it declared in base detail